.changeset/brave-pots-drop.md

```diff
@@ -0,0 +1,22 @@
+---
+'astro': minor
+---
+
+Page Partials
+
+Any page component can now be identified as a *partial*, allowing you fetch them in the client in order to replace only parts of the page. Partials are used in conjuction with a rendering library, like htmx or Stimulus or even just jQuery.
```

Comment:
```suggestion
A page component can now be identified as a *partial* page, which will render its HTML content without including a `<! DOCTYPE html>` declaration nor any `<head>` content.

A rendering library, like htmx or Stimulus or even just jQuery can access partial content on the client to dynamically update only parts of a page.
```
OK, the original is lost to history... but I was thinking of something *like* this. Again, edit as you like.